server/model/adjustmentsHubViewModel.ts

```diff
@@ -140,6 +140,31 @@ export default class AdjustmentsHubViewModel {
       const effective = adjustments.map(a => a.effectiveDays).reduce((sum, current) => sum + current, 0)
       return total - effective
     }
+
+    if (this.unusedDeductionMessage === 'UNSUPPORTED') {
+      const unusedDeductionAdjustmentDays =
+        this.adjustments.filter(it => it.source !== 'NOMIS').find(it => it.adjustmentType === 'UNUSED_DEDUCTIONS')
+          ?.effectiveDays || 0
+      const remandAdjustmentDays =
+        this.adjustments
+          .filter(it => it.adjustmentType === 'REMAND')
+          ?.map(it => it.days)
+          .reduce((acc, cur) => {
+            return acc + cur
+          }, 0) || 0
+      if (adjustmentType.value === 'REMAND') {
+        return unusedDeductionAdjustmentDays > remandAdjustmentDays
+          ? remandAdjustmentDays
+          : unusedDeductionAdjustmentDays
+      }
+
+      if (adjustmentType.value === 'TAGGED_BAIL') {
```

Comment:
This new block is not needed. The logic to display the number of unused days is exactly the same as above. You need to change the condition from

```
    if (this.unusedDeductionMessage === 'NONE') {
```

to

```
    if (this.unusedDeductionMessage === 'NONE' || (this.unusedDeductionMessage === 'UNSUPPORTED' && (unused deductions were entered in DPS))) {

```